### PR TITLE
Add blink components metadata to some content

### PR DESCRIPTION
**Pull request by ebidel**
Only did a few to start testing.

Easiest to review with https://github.com/google/WebFundamentals/pull/4796/files?w=1

### CODE REVIEW

File: src/content/en/fundamentals/getting-started/primers/async-functions.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ book_path: /web/fundamentals/_book.yaml
 description: Async functions allow you to write promise-based code as if it were synchronous
 
 {# wf_published_on: 2016-10-20 #}
-{# wf_updated_on: 2017-06-19 #}
```
Comment (sherwinjoseph) [inline src/content/en/fundamentals/getting-started/primers/async-functions.md]: what difference does this make

**Comment by kosamari** [comment]
Since this is adding metadata, rather than editing content, I'm inclined to _not_ update `wf_updated_on` since this date is visible on the site to see the up-to-date-ness of the content (version number/API change etc)

**Comment by ebidel** [comment]
I thought the gh bot complains when the file changes but the updated date
isn't modified?

**Comment by kosamari** [comment]
Yeah I think we need to change the warning message... it is a good reminder but not merge blocker in some case. (I've merged it without update before.) 
It's subjective what is "updated" mean though...

**Comment by ebidel** [comment]
Ah ok. For some reason I thought it prevented merges. If it's just a
warning, I'll revert the dates.

**Comment by ebidel** [comment]
PTAL

**Pull request merged**